src/topp/CometAdapter.cpp

```diff
@@ -285,12 +295,13 @@ class TOPPCometAdapter :
     num_enzyme_termini["N-term unspecific"] = 9;
 
     os << "search_enzyme_number = " << enzyme_number << "\n";                // choose from list at end of this params file
-    os << "num_enzyme_termini = " << num_enzyme_termini[getStringOption_("num_enzyme_termini")] << "\n";                  // 1 (semi-digested), 2 (fully digested, default), 8 C-term unspecific , 9 N-term unspecific
+    os << "search_enzyme2_number = " << enzyme2_number << "\n";              // second enzyme; set to 0 if no second enzyme
+    os << "num_enzyme_termini = " << num_enzyme_termini[getStringOption_("num_enzyme_termini")] << "\n"; // 1 (semi-digested), 2 (fully digested, default), 8 C-term unspecific , 9 N-term unspecific
     os << "allowed_missed_cleavage = " << getIntOption_("allowed_missed_cleavages") << "\n";             // maximum value is 5; for enzyme search
 
     // Up to 9 variable modifications are supported
-    // format:  <mass> <residues> <0=variable/else binary> <max_mods_per_peptide> <term_distance> <n/c-term> <required>
-    //     e.g. 79.966331 STY 0 3 -1 0 0
+    // # format:  <mass> <residues> <0=variable/else binary> <max_mods_per_peptide> <term_distance> <n/c-term> <required> <neutral_loss>
+    //     e.g. 79.966331 STY 0 3 -1 0 0 97.976896
```

Comment:
not sure if we should always add neutral losses

Reply:
but how do we get the possible losses for the different mods? is this registered somewhere?

Reply:
yep in residue db. but we only store one and in reallity it depends on the fragmentation method etc.